style.css

```diff
@@ -249,3 +247,66 @@ img {
 .item {
   flex-grow: 3;
 }
+
+.footer {
+  flex-direction: column;
+  display: flex;
+  background: #6070ff;
+  width: 100%;
+  border-radius: 40px 0 0 0;
+  padding: 146px 0;
+  margin-top: 50px;
+  color: #fff;
+  text-align: center;
+  height: 779px;
+  background-repeat: no-repeat;
+  background-image: url(bgn.svg);
+  background-position: right;
+}
+
+.footer form input,
+.footer form textarea {
+  width: 100%;
+  display: block;
+  height: 48px;
+  padding-left: 10px;
+  border-radius: 7px;
+  margin-top: 20px;
+  border: none;
+  background: #fff;
+}
+
+.footer form textarea {
+  height: 170px;
+}
+
+.footer h2 {
+  font-size: 48px;
+  margin-bottom: 25px;
+  height: 56px;
+}
+
+#final {
+  background-color: #fff;
+  margin: 17px;
+  width: 140px;
+  height: 48px;
+  font-weight: 500;
+  padding: 12px;
+  border-radius: 8px;
+  margin-left: 0;
+}
+
+#final button {
+  color: #6070ff;
+}
+
+#final:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+#final:active {
+  background-color: #2230d2;
+  color: white;
+}
```

Comment:
- [ ] I can see that you tried to implement the button interactions as the previous reviewer requested here, but it's not working as it should. the background color is changing but the button text color is not changing. I think that's because you are implementing it on the `<div>` container and not on the `<button>`. Kindly implement it on the button.
- [ ] [Optional] Also, since the button interactions are for all buttons and not just the form button, I will suggest you kindly create a class for all buttons and implement it on  or you just apply it to `<button>` tag

Reply:
thank you, bad mistake I just fixed it